packages/sui-studio/src/components/style/index.js

```diff
@@ -19,8 +19,12 @@ export default class Style extends Component {
     this._linkElement = createLinkElement()
   }
 
+  componentWillUnmount () {
+    const lastIndex = document.styleSheets.length - 1
+    document.styleSheets[lastIndex].disabled = true
+  }
```

Comment:
Another approach would be to run through all styleSheets searching for the same href, but as far as I know nothing else injects stylesheet in between. Let me know if this could be dangerously.